api/openapi_server/openapi/paths/auth/authResendConfirmationCode.yaml

```diff
@@ -0,0 +1,29 @@
+post:
+  description: Resends the registration confirmation code to the specified user (identified by email).
+  operationId: resend_confirmation_code
+  requestBody:
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            email:
+              type: string
+          required:
+            - email
+  responses:
+    '200':
+      description: successful operation
+      content:
+        application/json:
+          schema:
+            $ref: '../../openapi.yaml#/components/schemas/ApiResponse'
+    '400':
+      description: The email parameter was not sent or the user with the given email was not found.
+    '429':
+      description: Too many requests to resend the registration confirmation code were made to this user.
+  tags:
+    - auth
+  x-openapi-router-controller: openapi_server.controllers.auth_controller
+  security:
```

Comment:
I don't know if we want to require an auth token for this route since users may not be signed in when making the request. For example, post-sign up, but before signing in. I think the way Cognito is currently set up, a user without a verified email won't be able to sign in

Reply:
The auth token requirement has been removed.